internal/servicedeployer/compose.go

```diff
@@ -42,6 +42,8 @@ type DockerComposeServiceDeployerOptions struct {
 type dockerComposeDeployedService struct {
 	ctxt ServiceContext
 
+	shutdownTimeout string // default shutdown timeout 10 seconds
```

Comment:
Nit. Use `time.Duration` for this variable?

Reply:
Updated to use `time.Duration` :+1: 
I'll consider default value zero, as the timeout has not been set in that case. So it will use the default value from docker-compose 
https://docs.docker.com/compose/faq/#why-do-my-services-take-10-seconds-to-recreate-or-stop

Reply:
Updated here: https://github.com/elastic/elastic-package/pull/1727/commits/fd8321f920536ccb725eb727921578394505146b